node/src/chain_spec.rs

```diff
@@ -144,6 +146,8 @@ pub fn local_testnet_config() -> ChainSpec {
 						get_collator_keys_from_seed("Bob"),
 					),
 				],
+				// Sudo account
+				get_account_id_from_seed::<sr25519::Public>("Alice"),
```

Comment:
Here is a temporary Sudo account, which needs to be modified later.

Reply:
Why do we need to write a seed phrase in the chain config? It doesn’t make sense.

I just need a wallet addres to be a sudo. I shouldn’t need to give you my seed phrase.